website/docs/addon_motionbuilder_artist.md

```diff
@@ -0,0 +1,78 @@
+---
+id: addon_motionbuilder_artist
+title: Motionbuilder Docs
+sidebar_label: Motionbuilder
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+{versions.MotionBuilder_Badge}
+</ReactMarkdown>
+
+:::note Work in progress
+This part of documentation is still work in progress.
+:::
+## First Steps With AYON
```

Comment:
Would be nice to add short description what Ayon and MotionBuilder offers / allows to do within our integration...can be just few sentences. As me, myself not so sure what can be done with it...also noting version management among others.